ComplementaryData/modelCuration/FAEnewRxnMetAnnotation.tsv

```diff
@@ -0,0 +1,53 @@
+NewMetName	Charged formula 	Charge	compartment	KEGG ID	CHEBI ID	Remark
+ethyl hexanoate[m]	C8H16O2	0	mitochondrion		CHEBI:86055	
+ethyl hexanoate[c]	C8H16O2	0	cytoplasm		CHEBI:86055	
+ethyl hexanoate[e]	C8H16O2	0	endoplasmic reticulum		CHEBI:86055	
```

Comment:
Here you used `e` for endoplasmic reticulum but it should have been `er`, as `e` is reserved for "extracellular".

Reply:
Thanks @BenjaSanchez for catching that, but the [e] is correct. However, extracellular should be used instead of endoplasmic reticulum.